cmd/clusterctl/cmd/topology_plan.go

```diff
@@ -22,9 +22,9 @@ import (
 	"path"
 	"sort"
 
-	"github.com/google/go-cmp/cmp"
 	"github.com/olekukonko/tablewriter"
 	"github.com/pkg/errors"
+	"github.com/pmezard/go-difflib/difflib"
```

Comment:
The project seems not maintained anymore.
Could you point me to how it is managed diff in kubectl so I can take a look?

Reply:
kubectl makes use of an external `diff` binary via the `DiffProgram` struct:

https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/diff/diff.go#L213

Reply:
Ready to take a look implementation: https://github.com/kubernetes-sigs/cluster-api/compare/main...chrischdi:cluster-api:pr-topology-plan-diff-binary?expand=1

Reply:
I'm +1 to the proposed implementation that depends on diff command + allows override via an env variable
@vincepri opinions?

Reply:
+1 on the diff command dependency rather than depending on an unmaintained library

Reply:
changed implementation to adapt the `kubectl` style :-)